task-manager/src/components/NavbarComponent.vue

```diff
@@ -0,0 +1,124 @@
+                        >You have {{ remainingTasksCount }} tasks remaining
+                        today.</span
+                    >
+                    <span v-else>No remaining tasks for today.</span>
+                </v-tooltip>
+            </v-col>
+
+            <v-btn color="red lighten-1" @click="signOut">
+                <span>Sign Out</span>
+                <v-icon right>mdi-exit-to-app</v-icon>
+            </v-btn>
+        </v-app-bar>
+
+        <v-navigation-drawer
+            v-model="appConfig.drawer"
+            app
+            sticky
+            :temporary="appConfig.overlay && !appConfig.permanent"
+            :permanent="appConfig.permanent"
+            class="gray lighten-2"
+        >
+            <v-list-item class="py-1">
+                <v-list-item-avatar>
+                    <v-img :src="user.image"></v-img>
+                </v-list-item-avatar>
+
+                <v-list-item-title>{{ user.username }}</v-list-item-title>
+            </v-list-item>
+
+            <v-divider></v-divider>
+
+            <v-list rounded>
+                <v-list-item
+                    v-for="item in navItems"
+                    :key="item.title"
+                    router
+                    :to="item.route"
+                >
+                    <v-list-item-icon>
+                        <v-icon>{{ item.icon }}</v-icon>
+                    </v-list-item-icon>
+
+                    <v-list-item-content>
+                        <v-list-item-title>{{ item.title }}</v-list-item-title>
+                    </v-list-item-content>
+                </v-list-item>
+            </v-list>
+        </v-navigation-drawer>
+    </nav>
+</template>
+
+<script>
+export default {
+    data() {
+        return {
+            user: {
+                username: this.$store.getters.loggedUser.username,
+                image: this.$store.getters.loggedUser.image,
+            },
+            appConfig: {
+                drawer: false,
+                overlay: true,
+                permanent: true,
+            },
+            isMenuVisible: false,
+            navItems: {
+                dashboard: {
+                    title: "Dashboard",
+                    icon: "mdi-view-dashboard",
+                    route: "/dashboard",
+                },
+                taskManager: {
+                    title: "Task Manager",
+                    icon: "mdi-clipboard-check",
+                    route: "/tasks",
+                },
+                report: {
+                    title: "Report",
+                    icon: "mdi-chart-bar",
+                    route: "/report",
+                },
+            },
+        };
+    },
+    computed: {
+        remainingTasksCount() {
+            const today = new Date().toISOString().substring(0, 10);
+            return this.$store.getters.getTasksByLoggedUser.filter(
+                (task) => task.dueDate === today && !task.status
```

Comment:
when task is created status should be pending ....it should not be empty once completed its changed to completed. Edit the conditions accordingly

Reply:
I am using bool flag.